package.json

```diff
@@ -94,5 +94,8 @@
         "typedoc": "^0.23.26",
         "typedoc-plugin-markdown": "3.14.0",
         "typescript": "4.9.4"
+    },
+    "engines": {
+        "node": ">=18.0.0"
     }
```

Comment:
This field actually has an influence on the output Parcel produces and I was a bit wary of this causing a regression, so I compared the `yarn pack` output with and without this change. From skimming over the diff (annoyingly, all the module prefixes (e.g. `$hgUW1$createWriteStream`) also changed), as far as a I can tell the only significant change is that we're not using a `Buffer` polyfill anymore now:

![image](https://github.com/tweaselORG/appstraction/assets/4048809/2d1b60b4-318c-4a07-a289-874a3d37b45a)

That seems fine to me.